composer.json

```diff
@@ -30,7 +30,7 @@
         "malukenho/docheader": "^0.1.5",
         "mockery/mockery": "^0.9.5",
         "phpunit/phpunit": "^6.0.8 || ^5.7.15",
-        "zendframework/zend-coding-standard": "~1.0.0",
+        "zendframework/zend-coding-standard": "dev-feature/sniffs",
```

Comment:
I don't think it's a good idea to rely on development branches in stable packages.